### DISCUSSION TITLE
Boundary conditions for nested runs

### DISCUSSION THREAD

**Discussion by jwongala** [Parcels-code/Parcels#1364] [Q&A]
Hello! I am nesting two ROMS for the Hawaiian Islands ([Oahu Southshore](http://www.pacioos.hawaii.edu/metadata/roms_hiomsg.html) and the [MHI ROMS](http://www.pacioos.hawaii.edu/metadata/roms_hiig.html)). To deal with the boundaries, I implemented the partial slip boundary conditions, but I was curious what people's thoughts are about how particles are getting stuck on land. I attached an example of a particle trajectory. Any feedback is much appreciated! 
![Screenshot 2023-05-12 at 2 48 08 PM](https://github.com/OceanParcels/parcels/assets/36212394/a51d3a02-98d3-49aa-87ff-99b30bff0f56)

**Comment by jwongala** [comment]
Hello I wanted to follow up on my post and add the notebook I used. Any feedback is much appreciated! Thank you!
[TOOTS_oahuss_nesting_testing.txt](https://github.com/OceanParcels/parcels/files/11474645/TOOTS_oahuss_nesting_testing.txt)

**Reply by erikvansebille** [reply]
Thanks for your message, but what exactly is your question, @jwongala? It's difficult to give general feedback

**Comment by jwongala** [comment]
Hi @erikvansebille, Thanks for your response. My question would be, is it correct for the particles to be on land like this when I use the particle or freeslip boundary conditions? I have decided to use the method you all have of displacing particles when they are a certain distance from the coast for the time instead. But in combination with the other [issue](https://github.com/OceanParcels/parcels/issues/1362) I posted about nesting a regridded ROMS, I can only use the partial or freeslip boundary conditions to transport particles with the regridded ROMS nesting set up because particles aren't moving when I use the displacing boundary conditions.
